.github/workflows/appium-automation.yml

```diff
@@ -34,6 +34,7 @@ jobs:
         with:
           path: qa
           repository: BranchMetrics/qentelli-saas-sdk-testing-automation
+          ref: SDK-2465
```

Comment:
Is this still needed?

Reply:
Its pointing towards this branch/PR, but since i'm making Mobileboost tests that should be able to replace it in this case.